legacy/builder/setup_build_properties.go

```diff
@@ -126,6 +127,17 @@ func (s *SetupBuildProperties) Run(ctx *types.Context) error {
 
 	buildProperties.Merge(ctx.PackageManager.CustomGlobalProperties)
 
+	if !buildProperties.ContainsKey("build.keys.type") {
+		buildProperties.Set("build.keys.type", "public_keys") // The default is "pubblic_keys" for now
+	}
```

Comment:
```suggestion
```
This is not needed now. There is no use of the property, so it doesn't need to be set.

If it is later found that Arduino CLI must have special handling for multiple key types, some code of this sort should be added at that time in order to provide backwards compatibility for the existing platforms.